File: doc/Examples_SPL/expressionsCaesarJ/src/printInfix/Mult.java
cclass printInfix.PrintInfix;

public cclass Mult {
	/**
	 * Method to print a mult expression in infix format
	 */
	public void print(){
		System.out.print("(");
		op1.print();
		System.out.print("*");
		op2.print();
		System.out.print(")");	
	}//print

}

File: Alejandro/Sw/Benchmarks/PartialClassesExpressions/Expresiones/Infix/Mult.cs
using System;
using Expresiones;


namespace Expresiones
{
    public partial class Mult
    {
        
        /**
         * Method to print an expression in infix format
         * */
        public void print()
        {
            Console.Write("(");
            exp_izquierda.print();
            Console.Write("*");
            exp_derecha.print();
            Console.Write(")");
        }//print
    }//Mult
}//Expresiones